src/loaddb/load_server_loader.cpp

```diff
@@ -183,7 +183,7 @@ namespace cubload
 	  {
 	    heap_scancache_end (&thread_ref, &scancache);
 	    heap_attrinfo_end (&thread_ref, &attrinfo);
-	    m_error_handler.on_failure_with_line (LOADDB_MSG_LOAD_FAIL);
+	    m_error_handler.on_failure_with_line (LOADDB_MSG_LOAD_FAIL, er_msg ());
```

Comment:
`error_handler::on_failure_with_line` calls error_handler::log_error_message which already logs er_msg (), adding here as param again er_msg () might log the message twice.
I think it is enough to er_set ER_SM_ATTRIBUTE_NOT_FOUND at line 221